cmd/workflow-builder/main.go

```diff
@@ -32,6 +38,20 @@ func main() {
 		harborPass         = flag.String("harbor-pass", "", "The password for the harbor user.")
 	)
 
+	// if cluster is set, then
+	if cluster := os.Getenv("CLUSTER"); cluster != "" {
+		kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+	} else {
+		// If the home directory exists, then assume that the kube config will be read
+		// from ~/.kube/config.
+		if home := os.Getenv("HOME"); home != "" {
+			kubeconfig = flag.String("kubeconfig", filepath.Join(home, ".kube", "config"), "(optional) absolute path to the kubeconfig file")
+		} else {
+			// If the home directory doesn't exist, then allow the user to specify a path.
+			kubeconfig = flag.String("kubeconfig", "", "absolute path to the kubeconfig file")
+		}
+	}
```

Comment:
It might be nice to have support for the `KUBECONFIG` environment variable too.

Reply:
Good idea, I added support for it.